release-schema.json: add Prizes, PrizeDetail and lots.designContest (#2)
open-contracting-extensions/ocds_designContest_extension: README.md

# Design contest

Adds an object to the tender and lot objects to describe a design contest.

## Legal context

> In the European Union, this extension's fields correspond to [eForms BG-704 (Reward and Jury) and BG-44 (Prize)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/) and [Title III, Chapter II of Directive 2014/24/EU](https://eur-lex.europa.eu/legal-content/EN/TXT/?qid=1585836130257&uri=CELEX:32014L0024#d1e6612-65-1). See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).

Reviewer: ```suggestion
In the European Union, this extension's fields correspond to [eForms BG-704 (Reward and Jury) and BG-44 (Prize)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/) and [Title III, Chapter II of Directive 2014/24/EU](https://eur-lex.europa.eu/legal-content/EN/TXT/?qid=1585836130257&uri=CELEX:32014L0024#d1e6612-65-1). See [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/) for the correspondences to eForms fields. See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).

## Examples

> An example with a single prize for a tender.

Reviewer 2: ```suggestion
A contracting process with a single prize.
```

      ],
      "hasPrizes": true,
      "prizes": {

>         "description": "The winner(s), as well as each unsuccessful competitor who has provided services which meet the program, will receive, subject to the decision of the contracting authority, a prize of EUR 16,500.00 (VAT free).",

Reviewer 2: Should this be updated to reflect that there is only one prize in the `.details` array and to match the value of that prize?

```suggestion
        "description": "The winner will receive, subject to the decision of the contracting authority, a prize of EUR 3,000.00 (VAT free).",
```

Reviewer 2: Alternatively, you could leave the description as-is and omit the `.details` array from this example, since it is included in the lot example. In which case, you'll need to update the introductory sentence.

Author: good catch, those values should be the same but I don't think the sentence needs changed. My reading is that it's still a single prize even if multiple parties are awarded it as long as the value and prize reason/identity is the same. A bit like how 2 people can both win Gold medals in the 100m sprint if they both run exactly the same time. I'd consider that one prize awarded to 2 people rather than 2 separate prizes. So the `id` would be "Gold" and both winners get it if that makes sense?

Reviewer 2: Makes sense!

}
```

### Lot

> An example with multiple prizes for a single lot.

Reviewer 2: ```suggestion
A lot with multiple prizes.
```

```json
{
  "tender": {
    "lots": [
      {
        "id": "LOT-0001",
        "designContest": {
          "selectedParticipants": [
            {
              "id": "1",
              "name": "Brigitte Hermon"
            },
            {
              "id": "2",
              "name": "Paolo Travino"
            }
          ],
          "hasPrizes": true,
          "prizes": {
            "details": [
              {
                "id": "1",
                "description": "A lump sum indemnity of an amount identical to that paid to unsuccessful competitors will be paid to the winning team in the form of an advance on the project management contract at the end of the competition; this sum being credited to the amount of fees to be collected subsequently under the project management contract.",
                "value": {

>                   "amount": 3000,

Reviewer 2: Should this be the same value as the next prize in the `.details` array?

Author: No because they're two different prizes so can have different values

Reviewer 2: I find the description a bit confusing. It mentions an amount paid to unsuccessful competitors, is that reflected in the data anywhere?

Author: yes it is a bit confusing. It was just the description in the current version of the README, I'll simplify it

                "value": {
                  "amount": 10000,
                  "currency": "EUR"
                }
              }
            ]
          },
          "rewardsDetails": "The payment is made by administrative mandate within 30 days in accordance with the regulations in force.",
          "followUpContracts": true,
          "bindingJuryDecision": true,
          "juryMembers": [
            {
              "name": "Karla Schaffer"
            },
            {
              "name": "Bulat Kazinsky"
            }
          ]
        }
      }
    ]
  }
}
```

## Issues

Report issues for this extension in the [ocds-extensions repository](https://github.com/open-contracting/ocds-extensions/issues), putting the extension's name in the issue's title.

## Changelog

### 2023-03-10

> * Add `lots.designContest`, `Prizes` and `Prize` objects.

Reviewer: ```suggestion
* Add `lots.designContest` object.
* Add `details` array to `DesignContest.prizes` array, to describe the `id`, `description` and `value` of each prize.
```
